src/mixins/filing-template-mixin.ts

```diff
@@ -190,5 +199,10 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
 
     // Set Folio Number
     this.setFolioNumber(draftFiling.header.folioNumber)
+
+    // Set Cooperative Type if Coop
+    if (this.isTypeCoop) {
```

Comment:
Please move this into the switch statement above.

Reply:
Fixed in next commit.